src/config.c

```diff
@@ -86,15 +112,16 @@ static uint64_t _Config_GetVirtualKeyEntitiesThreshold(RedisModuleCtx *ctx,
 }
 
 void Config_Init(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
-	config.vkey_entity_count = _Config_GetVirtualKeyEntitiesThreshold(ctx, argv, argc);
-	config.thread_count = _Config_GetThreadCount(ctx, argv, argc);
+	config.vkey_entity_count = _Config_SetVirtualKeyEntitiesThreshold(ctx, argv, argc);
+	config.thread_count = _Config_SetThreadCount(ctx, argv, argc);
+	config.omp_thread_count = _Config_SetOMPThreadCount(ctx, argv, argc);
```

Comment:
whenever a configuration object is created for the first time it should populate all of it fields with default values, either constants or function calls.